packages/webapp/src/components/ImagePicker/index.tsx

```diff
@@ -42,6 +43,7 @@ type CommonProps = {
   label?: string;
   optional?: boolean;
   defaultUrl?: string;
+  disabled?: boolean;
```

Comment:
Should we call this `isDisabled` to make it consistent with the other changes?

Reply:
Do you know if there is a rule we can follow here?

I assumed `<Input />`, `<NumberInput />` , and `<RadioGroup />` all use `disabled` because they pass that prop eventually to a real html `<input />` (as does this component), and the ReactSelects use `isDisabled` because that's what the underlying ReactSelect expects.

I was trying to keep that, but I see now that we have about an equal amount of `disabled={isDisabled}` and `isDisabled={disabled}` in the codebase; so we appear all mixed up 😜

Reply:
That seems like a good way to reason it! There's no rules per se, and tbh I've seen similar mixes in every codebase I've seen 😅 I personally always prefer to do `is...` for boolean variables because I think it makes it clearer that they are booleans, but that's my preference (and I've probably not even followed my own rule more than once)